lib/basket.rb

```diff
@@ -0,0 +1,31 @@
+class Basket
+  def initialize(basket: [])
+    @basket = basket
+  end
+
+  def add(order)
+    @basket.append(order)
+  end
+
+  def viewBasket
+    return menu_string_constructor(@basket)
+  end
+
+  def basket
+    return @basket
+  end
+
+  def reciept
```

Comment:
Nice use of `inject`.  It's not the most intuitive method to use.  I think you could also use `sum`, which can take a block and will sum together all the return values of the block (without the need for an accumulator).